README.md

```diff
@@ -71,65 +71,37 @@ pip install -r requirements.txt
 
 ### Step 4: Configure the Environment
 
-1. Create a `.env` file in the project directory.
-2. Use the following template:
+1. Copy the `.env` and open it.
+
+```bash
+https://raw.githubusercontent.com/DoktorShift/naughtify/refs/heads/main/example.env
+mv example.env .env
+sudo nano .env
+```
+
+2. Fill in at least the first three fields of the template, e.g.:
```

Comment:
Fill in at least the first four fields of the template e.g.:


Telegram Bot Token
CHat ID (User ID)
LNBIts Readonly key
LNBits Server URL

These are heavily needed